### PR TITLE
[DE] Add `Countdown`

### PR DESCRIPTION
**Pull request by FreezyLemon**
I think I did a pretty good job on the `Countdown Offset` explanation. LMK what you think.

## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmap/Countdown/de.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+*Siehe auch: [Offset (Begriffsklärung)](/wiki/Offset)*
+
+Ein **Countdown** kann am Anfang einer [Beatmap](/wiki/Beatmap) erscheinen. Die meisten Beatmaps benötigen keinen Countdown, bei schnell oder abrupt startenden Songs kann er aber nützlich sein. Im `Design`-Tab des [Song-Setup-Fensters](/wiki/Client/Beatmap_editor/Song_setup) kann der Countdown an- und ausgeschaltet werden. In [osu!taiko](/wiki/Game_mode/osu!taiko)-Beatmaps darf kein Countdown verwendet werden.
+
+Die Countdown-Animation kann mithilfe der Buttons zur `Countdown Speed` (`Half`, `Normal`, `Double`) und mithilfe des Eingabefelds `Countdown Offset` angepasst werden. Die Buttons unter `Countdown Speed` können die Standardgeschwindigkeit der Countdown-Animation halbieren, beibehalten oder verdoppeln. Der Wert in `Countdown Offset` legt die Anzahl an zusätzlichen [Grundschlägen](/wiki/Music_theory/Tempo) fest, die zwischen dem Ende des Countdowns und dem ersten [Hit-Objekt](/wiki/Gameplay/Hit_object) eingefügt werden. Zum Beispiel startet bei einem `Countdown Offset` von 2 die Countdown-Animation 2 Grundschläge früher als sonst. Bei Maps mit Intros führt das Überspringen des Intros direkt zum Countdown.
```
Comment (The-Last-Cookie) [inline wiki/Beatmap/Countdown/de.md:15]:
```suggestion
Die Countdown-Animation kann mithilfe der Buttons zur `Countdown Speed` (`Half`, `Normal`, `Double`) und mithilfe des Eingabefelds `Countdown Offset` angepasst werden. Die Buttons unter `Countdown Speed` können die Standardgeschwindigkeit der Countdown-Animation halbieren, beibehalten oder verdoppeln. Der Wert in `Countdown Offset` legt die Anzahl an zusätzlichen [Grundschlägen](/wiki/Music_theory/Tempo) fest, die zwischen dem Ende des Countdowns und dem ersten [Hit-Objekt](/wiki/Gameplay/Hit_object) eingefügt werden. Zum Beispiel startet bei einem `Countdown Offset` von 2 die Countdown-Animation 2 Grundschläge früher als sonst. Bei Beatmaps mit Intros führt das Überspringen des Intros direkt zum Countdown.
```
I know that `Map` is pretty common, however, I have been avoiding this in the wiki and always used `Beatmap` instead.

File: wiki/Beatmap/Countdown/de.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Die Countdown-Animation kann [geskinnt](/wiki/Skinning/Interface#countdown) werden.
```
Comment (The-Last-Cookie) [inline wiki/Beatmap/Countdown/de.md:17]: This is pretty difficult to translate and I would like to make these suggestions:

```suggestion
Die Countdown-Animation ist [skinnbar](/wiki/Skinning/Interface#countdown).
```

The above was already used once in the wiki (also with its variation `skinbar`).

```suggestion
Die Countdown-Animation kann mit [Skins](/wiki/Skinning/Interface#countdown) versehen werden.
```

Also possible to add `benutzerdefinierten`/`eigenen` in between.

**Review by The-Last-Cookie — CHANGES_REQUESTED** [review]

File: wiki/Beatmap/Countdown/de.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+tags:
+  - timer
+  - Count-down
+---
+
+# Countdown
+
+*Für Turniertimer, siehe: [Befehle für das Turnier-Management](/wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands)*\
+*Für Vorschriften zu Countdowns, siehe: [Rankingkriterien § Allgemein § Richtlinien](/wiki/Ranking_criteria#guidelines)*\
```
[the remark this answers is not in the document]
Comment (FreezyLemon) [reply wiki/Beatmap/Countdown/de.md]: `Ranking-Kriterien` is an oversight, changed it. I think the reference can just be kept as-is for now.

File: wiki/Beatmap/Countdown/de.md (hunk 2 above)
Comment (FreezyLemon) [reply wiki/Beatmap/Countdown/de.md:17]: I don't like `mit Skins versehen werden.` very much, `ist skinnbar`. would be better. But is there a problem with `skinnen` as a verb? We do the same thing with `gerankte Beatmap`, it's even in the ASC.

Comment (The-Last-Cookie) [reply wiki/Beatmap/Countdown/de.md:17]: I will leave it to you what to use here. All three options are good in my opinion.

Comment (FreezyLemon) [reply wiki/Beatmap/Countdown/de.md:17]: I like my Countdown `geskinnt` ;)

**Review by Deimosu — APPROVED** [review]

**Review by The-Last-Cookie — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**